OxfordBodleian/Juel-Jensen/BDLaethe37.xml

```diff
@@ -158,100 +1601,230 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <foliation>
+                                    </foliation>
+                                    <collation>
+                                        <list>
+                                        <item xml:id="q11">
+                                            <dim unit="leaf">2</dim>
+                                            <locus from="94r" to="95v"/>
+                                        </item>
+                                        </list>
+                                    </collation>
+                                    <condition key="good"></condition>
+                                </supportDesc>
+                                
+                                <layoutDesc>
+                                    <layout columns="2" writtenLines="20">
+                                        <locus from="94ra" to="95rb"/>
+                                        <note corresp="#textarea2 #margin2">Data on margin dimensions taken from <locus target="#94r"/>.</note>
+                                        <!--text area dimensions missing!-->
+                                        <!-- <dimensions unit="mm" xml:id="textarea2">
+                                        <height></height>
+                                        <width></width>
+                                    </dimensions>-->
+                                        <dimensions type="margin" unit="mm" xml:id="margin2">
+                                            <dim type="top">11</dim>
+                                            <dim type="bottom">18</dim>
+                                            <dim type="right">14</dim>
+                                            <dim type="left">5</dim>
+                                            <dim type="intercolumn">4</dim>
+                                        </dimensions>
+                                    </layout>
+                                </layoutDesc>
+                                
+                            </objectDesc>
+                            <handDesc>
+                                <handNote xml:id="h2" script="Ethiopic">
+                                <locus from="94ra" to="95rb"/>
+                                <seg type="script"/>
+                                <seg type="ink">Black and red</seg>
+                                <date notBefore="1700" notAfter="1950"></date>
+                                <desc>
+                                    Irregular hand.
+                                </desc>
+                                <seg type="rubrication">
+                                    Words within the first line of textual units (<foreign xml:lang="gez">ለኪ</foreign>, 
+                                    <foreign xml:lang="gez">ማር</foreign>) on <locus target="#94r"/>; elements of certain 
+                                    punctuation marks on <locus target="#95rb"/>. Throughout, there are blank spaces
```

Comment:
Have we been treating this as rubrication systematically?

Reply:
We have noted in general "elements of punctuation marks" as a feature of rubrication where applicable, I believe